### PR TITLE
MultiCoordinateFrame helpers and TSRBM visualizer change

### PR DESCRIPTION
**Pull request by gizatt**
Adds some helpers to find frames within subframes, and cleans up the Atlas compile to be a bit cleaner. 

Also proposes changing the TSRBM visualizer input to be the output frame of the system it is created from, and not the state. (The visualizer is usually cascaded with the system, so the TSRBM output is what it takes as input anyway -- it has just been generally required that the state frame is available in that output somewhere.)

### CODE REVIEW

File: systems/frames/MultiCoordinateFrame.m [hunk 1]
```diff
@@ -293,10 +293,54 @@ function display(obj)
       fr = obj.frame{id};
     end
     
+    function fr = getFrameByNameRecursive(obj,name)
```
Comment (RussTedrake) [inline systems/frames/MultiCoordinateFrame.m]: i like these, but you're not actually using them anywhere... (unless I missed it?).  If we want them in, we should have at least a unit test so the code doesn't go stale..?

**Comment by RussTedrake** [comment]
otherwise looks good.

**Comment by gizatt** [comment]
Ah yes, good call. They're to be used in DRC code; I'll see if I bring over
a simplified version of that use case for a unit test.

**Comment by gizatt** [comment]
Added a new unit test

**Pull request merged**